### PR TITLE
fix/errors: Use thread's stderr for error output.

### PR DESCRIPTION
**Pull request by joshuef**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -87,10 +87,9 @@ pub fn run_with(cmd_args: Option<&[&str]>) -> Result<(), String> {
 
     // We need a minimum of INFO level for vaults verbosity,
     // since the genesis vault logs the contact info at INFO level
-    let verbosity = format!("-{}", "v".repeat(2 + args.vaults_verbosity as usize));
```
Comment (bochaco) [inline src/lib.rs]: We should still need this

File: src/lib.rs [hunk 2]
```diff
@@ -87,10 +87,9 @@ pub fn run_with(cmd_args: Option<&[&str]>) -> Result<(), String> {
...
     common_args.push(&verbosity);
 
-    // Construct genesis vault's command arguments
+    // Construct genesis vault's command argumen]ts
```
Comment (bochaco) [inline src/lib.rs]: minor typo here

File: src/lib.rs (hunk 1 above)
Comment (joshuef) [reply src/lib.rs]: Hmm, fat fingers as i worked or github's UI is doing some fun things. Will fix now

**Comment by joshuef** [comment]
Closing due to whackness. Will make a fresh one

**Pull request closed without merging**